### ISSUE TITLE
Missing document language

### ISSUE THREAD

**Issue by dotjay** [cis-bathnes/pattern-lib#10]
It looks like the default template in the UI toolkit is missing the language definition.
`<html lang="">`

Use the British region subtag variant for all templates:
`<html lang="en-GB">`

**Comment by BenDarby** [comment]
Added lang subtag to template files
